plugins/sslservices/sslservices.cpp

```diff
@@ -77,7 +77,7 @@ void failOpenSSLError(const std::string& context)
 int passphraseCB(char *passPhraseBuf, int passPhraseBufSize, int rwflag, void *pPassPhraseMB)
 {
     size32_t len = ((MemoryBuffer*)pPassPhraseMB)->length();
-    if (passPhraseBufSize >= (int)len)
+    if (passPhraseBufSize >= len)
```

Comment:
may need to cast passPhraseBufSize to size32_t to avoid complains about mixing signed and unsigned in a comparison